common/browser_utilities.py

```diff
@@ -12,15 +12,23 @@ def is_text_present_with_retry(text, retries=3, delay=1):
     return False
 
 
-def wait_for(fn, timeout_secs, retry_secs=3.0):
+def wait_for(fn, timeout_secs, retry_secs=3.0, argv=[]):
     """run function multiple times within a timeout window until it returns a truthy value
     Not done as a decorator as fn could be called from several places"""
 
     ret_val = None
     timeout = datetime.utcnow() + timedelta(seconds=timeout_secs)
 
     while not ret_val and datetime.utcnow() < timeout:
-        ret_val = fn()
+        ret_val = fn(argv) if argv else fn()
         if not ret_val:
             time.sleep(retry_secs)
     return ret_val
+
+
+def named_element_on_page(name):
```

Comment:
Doc string explaining what it's for and what the variables are used for

Reply:
Done